src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/RiskConsumer.swift

```diff
@@ -18,6 +18,12 @@
 //
 
 import Foundation
+import UIKit
+
+/**
+An instance of RiskConsumer is an object which is interesting in the result of risk calculation.
```

Comment:
typo?
https://dictionary.cambridge.org/de/worterbuch/englisch/interesting
https://dictionary.cambridge.org/de/worterbuch/englisch/interested
```suggestion
An instance of RiskConsumer is an object which is interested in the result of risk calculation.
```